gordo_components/builder/mlflow_utils.py

```diff
@@ -257,7 +257,12 @@ def get_batch_kwargs(metadata: dict) -> dict:
 
     metric_list: List[Metric] = list()
 
-    # Parse cross-validation metrics
+    # Parse cross-validation split metadata
+    if metadata["model"].get("cross-validation", {}).get("splits", None):
+        splits = metadata["model"]["cross-validation"]["splits"]
```

Comment:
I _think_ this will always result in `None` because the `cross-validation` is stored under `build-metadata -> model`, no?

Reply:
As per comment below, I'll delay this one for #795 as it's fixed there.

Reply:
Probably also a separate issue that can be discussed. But after some reading about how to best retrieving/checking for sub-keys from a nested dictionary, people say that using try/catch might be better than using the .get().get() thingy. It's easy to shoot oneself in the foot, if one of the get's return something else than a dict (or something that cannot be called get() on). 
Perhaps we should think about this at least. If the structure does not change, and we are guaranteed to always nest it in this way, then we will probably not have an issue, but it might be more error prone if things change.

Reply:
Nice :+1: All that said, I think generalizing this method once some dust has settled and we are sure we are keeping all of this would be prudent and wise.

Reply:
I decided to go with Einar's suggested here on this go around, since I needed to merge these changes anyway :+1: